### DISCUSSION TITLE
"Codespaces is rolling out progressively but isn’t available in this organization yet." ?

### DISCUSSION THREAD

**Discussion by ShonFrazier** [community/community#5062] [Codespaces] [answered]
I upgraded my org to a team yesterday (12 Aug) to get access to Codespaces. But when I go to organization settings, I see this message (in the subject). I'm able to use Codespaces on my _personal_ repos, however. Is this intended behavior for paid organization accounts?

**Comment by bailey** [comment]
Hi there @ShonFrazier! You are a part of the individual beta, so that will be a separate experience versus enabling your Teams plan organization. Can you share the newly upgraded organization name with me and I can look into this?

**Reply by ShonFrazier** [reply]
The organization is “Fyrestead, LLC”

P.S. My emailed reply didn't appear as a reply to your message as I expected, so I moved my reply.

**Reply by lewallen** [reply]
Seconded. I upgraded my org ("mathemaddicts") to a paid team, and the codespaces settings page says "Codespaces is rolling out progressively but isn’t available in this organization yet." @bailey If this is a mistake, any advice posted here would be appreciated. If not it would still be nice to understand the situation.

**Reply by bailey** [reply]
@ShonFrazier is https://github.com/Fyrestead the org you're referencing?

**Reply by ShonFrazier** [reply]
Yes, that's the one

**Reply by bailey** [reply]
@ShonFrazier @lewallen both of those orgs should now be good to go! check out your org settings to get codespaces enabled. let me know if you have any other problems ✨

**Reply by lewallen** [reply]
@bailey Thanks very very much!!

**Reply by ShonFrazier** [reply]
@bailey awesome - thanks for the help!

**Comment by dewinterjack** [comment]
The announcement was very misleading.

> Codespaces now available for GitHub Teams and GitHub Enterprise Cloud

This would make anyone think that upgrading to the Teams plan would grant immediate access to Codespaces for an organization. 
The documentation for Codespaces literally has the announcement posted at the top: 

<img width="1472" alt="Screenshot 2021-08-14 at 12 22 31" src="https://user-images.githubusercontent.com/17269242/129444787-4dc53181-d9fd-4a4b-901a-cceb86b9e578.png">

This really should be changed so more people end up making a misleading purchase.

@bailey The org name is better-experience

**Reply by MightySCollins** [reply]
Yep we upgraded `Cyberbyte-Studios` to teams to get it and was slightly disappointed. When looking I could not even see a note in the docs saying its in still being rolled out. I understand why new orgs should have a delay before they can spin up vm's but it's not really clear.

**Reply by bailey** [reply]
Hey there @dewinterjack and @MightySCollins! Apologies as these more recent org upgrades have required manual enablement in order to mitigate abuse. I've pushed through both of these organizations so you should both be good to go now!

**Reply by bailey** [reply]
We are working on this experience and will get something more informative in-product ASAP ✨

**Reply by cl1ent** [reply]
@bailey I've encountered the same issue and just upgraded to the teams plan to test codespaces. Any chance you could enable the feature for our org `systemiqofficial` (https://github.com/systemiqofficial)?
Would be very much appreciated!

EDIT: sending an email to codespaces@github.com helped

**Reply by Nautman** [reply]
@bailey, if you are authorized to enable codespaces for my organisation `namequiz`, https://github.com/namequiz, I would very much appreciate if you could enable it :)

**Reply by cl1ent** [reply]
@Nautman might be quicker if you send an email to `codespaces@github.com` and ask them to enable it. worked for me ;)

**Reply by Nautman** [reply]
Thanks a lot for the tip! Will do :) @cl1ent

**Comment by tanmayeekamath** [comment, marked as answer]
Hey everyone, please reach out to `codespaces@github.com` if you would like to enable Codespaces for a newly created/upgraded Team/Org. 

Thank you for your patience and feel free to reach out for any questions on this forum :)
